Workflow.cpp

```diff
@@ -119,6 +120,31 @@ void Workflow::rewriteMethodCall(LLILFunctionRef ssa, size_t insnIndex)
     llil->GenerateSSAForm();
 }
 
+void Workflow::rewriteCFString(LLILFunctionRef ssa, size_t insnIndex)
+{
+    const auto bv = ssa->GetFunction()->GetView();
+    const auto llil = ssa->GetNonSSAForm();
+    const auto insn = ssa->GetInstruction(insnIndex);
+    const auto llilIndex = ssa->GetNonSSAInstructionIndex(insnIndex);
+    auto llilInsn = llil->GetInstruction(llilIndex);
+
+    auto sourceExpr = insn.GetSourceExpr<LLIL_SET_REG_SSA>();
+    auto destRegister = llilInsn.GetDestRegister();
+
+    auto addr = sourceExpr.GetValue().value;
+    auto stringPointer = addr + 0x10;
```

Comment:
What is this magic number? Does it change depending on architecture?

Reply:
It is an offset into the structure for the type. workflow_objc currently only targets 64 bit archs, so this will always be the same.

Properly having it support 32 bit platforms ( #11 ) is a very large task involving setting up a ton of predefined structures to start loading things into, involves digging into armv7 objc internals, etc. This one will get cleaned up in that large pass.